telemetry/jetbrains/src/test/resources/testGeneratorAllowedValuesOutput

```diff
@@ -0,0 +1,101 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+// THIS FILE IS GENERATED! DO NOT EDIT BY HAND!
+@file:Suppress("unused", "MemberVisibilityCanBePrivate")
+
+package software.aws.toolkits.telemetry
+
+import com.intellij.openapi.project.Project
+import java.time.Instant
+import kotlin.Boolean
+import kotlin.Double
+import kotlin.String
+import kotlin.Suppress
+import kotlin.Unit
+import software.aws.toolkits.core.ConnectionSettings
+import software.aws.toolkits.jetbrains.services.telemetry.MetricEventMetadata
+import software.aws.toolkits.jetbrains.services.telemetry.TelemetryService
+
+/**
+ * A test object for parsing allowedValues
+ */
+public enum class TestAllowedValues(
+    private val `value`: String,
+) {
+    TestSpacesAreReplaced("test spaces are replaced"),
```

Comment:
Key is converted, but NOT the value. In general the advice was to avoid spaces for values.

Reply:
I would  expect this not be a breaking change, since no other `allowedValues` have spaces other than the values my team introduced this week.